### DISCUSSION TITLE
Courseplay auf Didiserver

### DISCUSSION THREAD

**Discussion by OnkelBuck** [Courseplay/Courseplay_FS22#1269] [Q&A] [answered]
Hallo
Was muss man tun das alle die dem spiel beitreten die gespeicherten Kurse sehen bzw. auch welche speichern können ?
Die Kurse hab ich zwar im LS22-Ordner unter modsetting liegen  aber nicht auf dem Server so ich nur ich die Kurse.
Im LS19 lagen die nämlich auch mit auf dem Server.

**Comment by Tensuko** [comment, marked as answer]
Was man tun muss? Abwarten!
Kurse können die User speichern wenn sie in ein Fahrzeug springen, das einen Kurs hat. Der sollte synchronisiert sein und kann dann von jedem abgespeichert werden.

Kurse auf der Server ist so noch nicht vorgesehen.

**Reply by Tobi25778** [reply]
Gibt es hier bereits ein Update?
Bzw. ist es in Planung, dass die erstellten Kurse in einem Ordner auf dem DeDi Server gespeichert werden?

**Reply by Tensuko** [reply]
Bis her nicht, nein.

**Reply by Tobi25778** [reply]
Das ist schade.
Besteht denn Hoffnung, dass dieses Feature kommen wird?

Seitdem das Internet immer schneller und stabiler wird, werden auch die Communities mit Servern mehr,

**Reply by Tensuko** [reply]
Naja liegt leider nicht an der Internetgeschwindigkeit usw, sondern eher daran wie sich das synchronisierne lässt :)

Müssen wir uns anschauen ob das klappt.

Eventuell hat @schwiti6190 ja doch noch ne Idee dafür.

**Reply by Tobi25778** [reply]
Ne Idee wäre, dass der Standartordner halt der auf dem DediServer ist, aber müssten dann ja die Login Daten vom FTP Server mit hinterlegt sein, daran könnte es hapern.

**Reply by Tensuko** [reply]
Ne das funktioniert komplett anders.
Macht es leider nicht einfacher.... müssen wa schauen.

**Comment by OnkelBuck** [comment]
Aha alles klar  danke
